modules/dxp/apps/analytics/analytics-reports-journal-impl/src/main/java/com/liferay/analytics/reports/journal/internal/info/item/JournalArticleAnalyticsReportsInfoItem.java

```diff
@@ -149,35 +228,30 @@ public String getTitle(JournalArticle journalArticle, Locale locale) {
 
 	@Override
 	public boolean isShow(JournalArticle journalArticle) {
-		Layout layout = _getLayout(journalArticle);
-
-		if (layout == null) {
-			return false;
-		}
-
-		if (!layout.isTypeAssetDisplay()) {
-			return false;
-		}
-
-		if (_isEmbeddedPersonalApplicationLayout(layout)) {
-			return false;
-		}
-
-		try {
-			if (!_hasEditPermission(
-					journalArticle, layout,
-					PermissionThreadLocal.getPermissionChecker())) {
-
-				return false;
+		Optional<Layout> layoutOptional = _getLayoutOptional(journalArticle);
+
+		return layoutOptional.filter(
+			layout -> layout.isTypeAssetDisplay()
```

Comment:
This lambda can be replaced with method reference, right?

Reply:
yeahh!!